sbnci/Common/Modules/TPCReco/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+if(SBND)
+add_subdirectory(NoiseTest)
+add_subdirectory(dEdx)
+add_subdirectory(RecoEff)
```

Comment:
the `if(SBND)` line is just a stop-gap measure until we can get those modules working for ICARUS. I'm optimistic this can be done, so I would we should keep these where they are for now.

Reply:
That's totally fair!